src/site/layouts/health_care_region_detail_page.drupal.liquid

```diff
@@ -5,79 +5,149 @@
 <div class="interior" id="content">
   <main class="va-l-detail-page va-facility-page">
     <div class="usa-grid usa-grid-full">
-      {% if entityUrl.path == "/outreach-and-events" %}
-        {% include 'src/site/navigation/facility_sidebar_nav.drupal.liquid' with sidebarData = outreachSidebar %}
-      {% else %}
-        {% include 'src/site/navigation/facility_sidebar_nav.drupal.liquid' with sidebarData = facilitySidebar %}
-      {% endif %}
-      <div class="usa-width-three-fourths">
-       {% include "src/site/components/navigation-sidebar-trigger.html" with menuId = "va-detailpage-sidebar" %}
-        <article aria-labelledby="article-heading" role="region" class="usa-content">
-          <h1 id="article-heading">{{ title }}</h1>
-          <div class="va-introtext">
-            <p>{{ fieldIntroText }}</p>
-          </div>
+      {% if entityUrl.path contains "/safety-security/" and buildtype != 'vagovprod' %}
+        {% include "src/site/components/navigation-sidebar-trigger.html" with menuId = "va-detailpage-sidebar" %}
```

Comment:
If we don't have the sidebar on these pages, do we still need the trigger?

Reply:
I actually don't know what the `navigation-sidebar-trigger` is so I just left it. Happy to remove it though if you are confident we don't need it! :)

Reply:
From https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/components/navigation-sidebar-trigger.html#L7:

> Used to open or close the side bar navigation bar on mobile screens. Must be used in a template with navigation-sidebar.html

I'm not super familiar with it either - thought you might know why it was there so I thought I'd ask :upside_down_face:

Reply:
It was a good catch! Updated! 👍